lib/ide/utils/src/handle.rs

```diff
@@ -0,0 +1,268 @@
+//! This module implements utilities that allow for a safer usage pattern of
```

Comment:
This file path does not tell what it is about. It should be `lib/ide/utils/src/cell/handle.rs`

Reply:
i went with just `cell`, so we have cell::WeakHandle (no handle::Handle)